src/ansiblelint/schemas/changelog.json

```diff
@@ -18,31 +46,69 @@
               },
               "type": "array"
             },
+            "known_issues": {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            },
+            "major_changes": {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            },
             "minor_changes": {
               "items": {
                 "type": "string"
               },
               "type": "array"
             },
             "release_summary": {
+              "markdownDescription": "This must be valid [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText).",
```

Comment:
This also applies to every list entry in the other sections.

Reply:
I am not going to copy the text in each section. There is no way to inherit it.

Reply:
I would have assumed you could do something like `"$ref": "#/$defs/changelog-section"` (for a generic 'list' section).